scripts/claude_mcp.sh

```
@@ -78,102 +153,1467 @@ log_error_details() {
+        log_with_timestamp "Server github-server already exists, skipping"
+        INSTALL_RESULTS["github-server"]="ALREADY_EXISTS"
+        SUCCESSFUL_INSTALLS=$((SUCCESSFUL_INSTALLS + 1))
+    else
+        echo -e "${BLUE}  🔗 Adding GitHub official remote MCP server...${NC}"
+        log_with_timestamp "Adding GitHub official remote MCP server"
+
+        # Remove any old deprecated GitHub server first
+        claude mcp remove "github-server" >/dev/null 2>&1 || true
+
+        # Add the new official GitHub HTTP MCP server (secure: token via temporary file)
+        local temp_config=$(mktemp -t github_mcp.XXXXXX)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)